src/L2TokenBridge.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
So I guess it is preferred to keep the duplication in favor to avoid inheritance. Will resolve this excepting we want to discuss something more. cc @telome